File: DepartmentAutomation.Application/Common/Models/WordDocument/CompetenceFormationLevel.cs
using System.Collections.Generic;
using DepartmentAutomation.Application.Common.Mappings;
using DepartmentAutomation.Domain.Enums;

namespace DepartmentAutomation.Application.Common.Models.WordDocument
{
    public class CompetenceFormationLevel : IMapFrom<Domain.Entities.CompetenceInfo.CompetenceFormationLevel>
    {
        public int LevelNumber { get; set; }

        public FormationLevel FormationLevel { get; set; }

        /// <summary>
        ///     Содержательное описание уровня
        /// </summary>
        public string FactualDescription { get; set; }

        /// <summary>
        ///     Результаты обучения
        /// </summary>
        public string LearningOutcomes { get; set; }

        public List<EvaluationToolType> EvaluationToolTypes { get; set; }
    }
}

File: ClientApp/src/app/models/Response/CompetenceFormationLevel.ts
import { FormationLevel } from '../Enums/FormationLevel';
import { EvaluationToolType } from './EvaluationToolType';

export interface CompetenceFormationLevel {
  id: number;
  formationLevel: FormationLevel;
  factualDescription: string;
  learningOutcomes: string;
  evaluationToolTypes: EvaluationToolType[];
}
